modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1231 @@
+//#TINY-9432
+
+Previously, if a _Mention_ was added to a {productname} editor; and the _Mention_ was selected; copied or cut; and then pasted on to a new line, attempting to then create a new line by pressing the **Return** key failed.
+
+As well, and depending on the host browser, the insertion point would not present in the editor until it was moved, by clicking elsewhere in the editor.
+
+This could be worked-around by moving the insertion point and then moving it back to immediately after the _Mention_ or by typing any character and then deleting that character.
+
+With this update, pressing **Return** immediately following the pasting of a _Mention_ string in a {productname} editor creates a new line, as expected.
+
+Also, the insertion point displays in all browsers immediately following a pasted _Mention_ string, also as expected.
+
+For information on the **Mentions** plugin, see xref:mentions.adoc[Mentions].
+
+
+=== Spell Checker Pro 3.2.1
+
+The {productname} 6.4 release includes an accompanying release of the **Spell Checker Pro** premium plugin.
+
+**Spell Checker Pro** 3.2.1 includes the following fix.
+
+==== Spell Checker Pro plugin over-loaded the spell checking service
+//#TINY-9391
+
+Previously, the Spell Checker Pro plugin would request each word in the content, and repeatedly request these words every time the spellchecker was activated. This resulted in duplicate words being requested during regular use of the plugin.
+
+With this update, Spell Checker Pro now only sends requests to the spelling service for each unique word and caches these results for later retrieval. This greatly reduces the amount and size of requests sent by the plugin during regular use, improving the reliability of the service and the plugin.
+
+For information on the **Spell Checker Pro** plugin, see: xref:introduction-to-tiny-spellchecker.adoc[Introduction to Spell Checker Pro].
+
+
+=== Advanced Code Editor 3.2.1
+
+The {productname} 6.4 release includes an accompanying release of the **Advanced Code Editor** premium plugin.
+
+**Advanced Code Editor** 3.2.1 includes the following improvements and additions.
+
+==== New button to copy the current dialog content.
+
+This update adds a new **Copy code** button to the Advanced Code Editor dialog, enabling the copying of the entire contents of the dialog without having to first select.
+
+==== New buttons to increase and decrease the font size in the dialog.
+
+This update adds two new buttons for increasing and decreasing the font size of code displayed in the Advanced Code Editor dialogs.
+
+==== New button to toggle dark mode.
+
+This update adds a new **Dark/light mode** button for toggling the Advanced Code Editor dialog between dark and light display modes.
+
+==== New button to toggle fullscreen.
+
+This update adds a new **Fullscreen** button that for toggling the Advanced Code Editor dialog between fullscreen and dialog modes.
```

Comment:
```suggestion
This update adds a new **Fullscreen** button for toggling the Advanced Code Editor between fullscreen and dialog modes.
```
Copy-edits.
1. s/that//.
2. s/Editor dialog/Editor/.

Reply:
@brian-forte closing this suggestion due to new content suggestions.